src/securityPolicy.c

```diff
@@ -363,13 +366,17 @@ security_policy_t policyForSignTxWitness(
 	DENY_UNLESS(is_valid_witness(pathSpec));
 
 	if (isSigningPoolRegistrationAsOwner) {
+
 		DENY_UNLESS(is_valid_stake_pool_owner_path(pathSpec));
+
+		WARN_UNLESS(bip44_hasReasonableAccount(pathSpec));
```

Comment:
I would just do `WARN_UNLESS(has_reasonable_account_and_address(pathSpec));` for both branches as we apply this logic for staking paths if isSigningPoolRegistrationAsOwner is false and I see no reason to do something different for this usecase, even though the address index check is technically redundant, but I would opt for consistency and simplicity here